ui/apps/dashboard/src/components/Onboarding/InvokeFn.tsx

```diff
@@ -202,7 +202,7 @@ export default function InvokeFn() {
               />
               <Button
                 appearance="outlined"
-                label="Skip, take me to dashboard"
+                label="Skip; go to dashboard"
```

Comment:
Can we replace ";" with either a "," or change the copy to "Skip. Go to dashboard" ?

Reply:
What about "Skip to dashboard"?